app/src/main/java/kr/co/connect/boostcamp/livewhere/di/appModules.kt

```diff
@@ -0,0 +1,13 @@
+package kr.co.connect.boostcamp.livewhere.di
+
+
+val appModules = arrayListOf(
+    apiModule
+    , loginModule
+    , sharedModule
+    , databaseModule
+    , homeModule
+    , mapModule
+    , reverseGeoApiModule
+    , detailModule
```

Comment:
```suggestion
    apiModule,
    loginModule,
    sharedModule,
    databaseModule,
    homeModule,
    mapModule,
    reverseGeoApiModule,
    detailModule
```
P3. 포메팅을 위와 같이 해주시면 좋습니다.